script.js

```diff
@@ -149,9 +149,63 @@ function addCheckboxes() {
 		checkbox = document.createElement('input');
 		checkbox.type = 'checkbox';
 		checkbox.value = sha;
+		checkbox.onclick = highlightRange;
 		checkbox.className = 'commit-compare-checkbox';
 		
 		listItem.appendChild(checkbox);
+
+		function highlightRange() {
+			var allCheckboxes = getAllCheckboxElements(),
```

Comment:
`allCheckboxes ` could possibly be made "global variable" so `getAllCheckboxElements()` is not needed to be called again here. Minor performance improvement